core/src/main/scala/com/nvidia/spark/rapids/tool/ToolTextFileWriter.scala

```diff
@@ -37,7 +37,7 @@ class ToolTextFileWriter(
   val LOG_FILE_PERMISSIONS = new FsPermission(Integer.parseInt("660", 8).toShort)
   val LOG_FOLDER_PERMISSIONS = new FsPermission(Integer.parseInt("770", 8).toShort)
   private val textOutputPath = new Path(s"$finalOutputDir/$logFileName")
-  private val hadoopConfToUse = hadoopConf.getOrElse(RapidsToolsConfUtil.newHadoopConf)
+  private lazy val hadoopConfToUse = hadoopConf.getOrElse(RapidsToolsConfUtil.newHadoopConf)
```

Comment:
does this have to be lazy? meaning we don't want it to be constructed until something else is initialized?  If so we may put a comment here

Reply:
It is not necessary. Removed.